tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -62,4 +65,90 @@ describe('Test Fleet access with RBAC with custom roles', { tags: '@rbac' }, ()
+        rules: [
+          { resource: "fleetworkspaces", verbs: ["list", "create"]},
+          { resource: "gitrepos", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+          { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        ]
+      });
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName)
+
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // What the user should be able to access
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.wait(500)
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Bundles');
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Workspaces');
+      cy.verifyTableRow(0, 'Active', 'fleet-default');
+      cy.verifyTableRow(1, 'Active', 'fleet-local');
+      cy.get('a.btn.role-primary').contains('Create').should('be.visible');
+      
+      // What the user should NOT be able to access
+      cy.open3dotsMenu('fleet-default', 'Delete', true);
+      cy.open3dotsMenu('fleet-default', 'Edit Config', true);
+    })
+  )
+
+  qase(44,
+    it('Test "Std-Base" user with custom role to "fleetworkspaces" all verbs except "delete can "edit" but not "delete" them', { tags: '@fleet-44' }, () => {
+      
+      const stduser = "std-user-44"
+      const customRoleName = "fleetAllExceptDeleteFleetWorkspaces"
+
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      cy.createRoleTemplate({
+        roleType: "Global",
+        roleName: customRoleName,
+        rules: [
+          { resource: "fleetworkspaces", verbs: ["create", "get", "list", "patch", "update", "watch"]},
+          { resource: "gitrepos", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+          { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        ]
+      });
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName)
+
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // What the user should be able to access
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.wait(1000)
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Bundles');
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Workspace');
+      cy.get('a.btn.role-primary').contains('Create').should('be.visible');
+      cy.verifyTableRow(0, 'Active', 'fleet-default');
+      cy.verifyTableRow(1, 'Active', 'fleet-local');
+      cy.open3dotsMenu('fleet-default', 'Edit Config');
+      cy.contains('allowedTargetNamespaces').should('be.visible');
+      
+      // What the user should NOT be able to access (once in Fleet Workspace menu)
```

Comment:
Comment is unclear :thinking:

Reply:
What is unclear?
It points to what the user should not be able to access

Reply:
`verbs` word is not in that comment..

Reply:
Like: // What verb(s) user should NOT be able to access (once in Fleet Workspace menu)

Reply:
I thought it was clear enough by the title pointing that should not be able to delete and by next command:
```
cy.open3dotsMenu('fleet-default', 'Delete', true);
```

But ok will change to this:

```
// Ensuring the user is not able to "delete" workspaces. 
```